### PR TITLE
Add CHANGELOG and version number

### PR DESCRIPTION
**Pull request by mkevinosullivan**
### WHY are these changes introduced?

In order to aide post-launch support, we should include a version number and a CHANGELOG file.

### WHAT is this pull request doing?

- Adding a `CHANGELOG` file
- Update the version number in the `package.json` file
- Add a checklist to the pull request template

### CODE REVIEW

**Review by paulomarg — APPROVED** [review]
Just a nit / question!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -20,3 +20,12 @@ Fixes #0000 <!-- link to issue if one exists -->
   Summary of the changes committed.
   Before / after screenshots appreciated for UI changes.
 -->
+
+### Checklist
+
+**Note**: once this PR is merged, it becomes a new release for this template.
+
+- [ ] I have added/updated tests for this change
+- [ ] I have incremented the version in the top-level `package.json` file **AND** in the `web/version.txt` file
```
Comment (paulomarg) [inline .github/PULL_REQUEST_TEMPLATE.md:29]: Do we still need to follow the top-level version?

**Review by paulomarg — APPROVED** [review]
LGTM, just a general question.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (mkevinosullivan) [reply .github/PULL_REQUEST_TEMPLATE.md:29]: I think it's probably good practice to include it in the package.json too, no?

**Pull request merged**